concordium-consensus/src/Concordium/ImportExport.hs

```diff
@@ -1,8 +1,13 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
```

Comment:
Please don't use this extension. It makes for difficult errors. Use injective type families instead, see, e.g., UpdatableContractState